### DISCUSSION TITLE
Is it possible to import Cannon ES just like Cannon JS

### DISCUSSION THREAD

**Discussion by AryaaSk** [pmndrs/cannon-es#149] [Q&A]
I have been using CannonJS, but recently I have realised that there are some issues and the library has been unmaintained for many years, so I found cannon-es.

It looks good but I need to be able to import it using a script tag, not an NPM install.

In CannonJS you can just import it like this:
```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/cannon.js/0.6.2/cannon.js"></script>  

<script>
//And then just use like this
const world = new CANNON.World();
</script>
```

Is there a way to import Cannon-Es the same way ?

**Comment by marcofugaro** [comment]
yes of course! you can import it like this:

```html
<script type="module">
import * as CANNON from 'https://unpkg.com/cannon-es@0.19.0/dist/cannon-es.js'

// And then just use like this
const world = new CANNON.World();
</script>
```
